internal/downloader/downloader.go

```diff
@@ -53,6 +57,14 @@ func Download(ctx context.Context, destDir string, sourceUrl string, showMsg boo
 		return fmt.Errorf("attempting to download from insecure source: %s", sourceUrl)
 	}
 
+	// read the USEGOGATHER env variable to determine which downloader to use:
+	// - if USEGOGATHER=1, use the go-gather downloader
+	// - if USEGOGATHER is not set, or set to any other value, use the Conftest downloader
+	if os.Getenv("USEGOGATHER") == "1" {
```

Comment:
You could make it a little tidier by moving this to `internal/utils/helpers.go`, similar to `utils.Experimental()` defined in there.

Reply:
Actually I'm confused about why lines 60-63 are needed. Wouldn't it be better to keep going to line 80 before checking the env var? That way the `WithDownloadImpl` used in testing to substitute in a mock downloader could still work.

Or, if not, then how would we ever get to line 80?

Reply:
Also, we'd want the logging on line 68-72 to be done either way I think.

Reply:
This was tidied up in a subsequent push.